src/vs/workbench/services/issue/browser/issueService.ts

```diff
@@ -5,19 +5,42 @@
 
 import * as dom from 'vs/base/browser/dom';
 import { userAgent } from 'vs/base/common/platform';
-import { IExtensionDescription } from 'vs/platform/extensions/common/extensions';
-import { IssueReporterData } from 'vs/platform/issue/common/issue';
+import { IExtensionDescription, ExtensionType } from 'vs/platform/extensions/common/extensions';
 import { normalizeGitHubUrl } from 'vs/platform/issue/common/issueReporterUtil';
+import { getZoomLevel } from 'vs/base/browser/browser';
+import { mainWindow } from 'vs/base/browser/window';
+import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
+import { IExtensionManagementService } from 'vs/platform/extensionManagement/common/extensionManagement';
+import { InstantiationType, registerSingleton } from 'vs/platform/instantiation/common/extensions';
+import { IIssueMainService, IssueReporterData, IssueReporterExtensionData, IssueReporterStyles } from 'vs/platform/issue/common/issue';
 import { IProductService } from 'vs/platform/product/common/productService';
-import { IExtensionService } from 'vs/workbench/services/extensions/common/extensions';
+import { buttonBackground, buttonForeground, buttonHoverBackground, foreground, inputActiveOptionBorder, inputBackground, inputBorder, inputForeground, inputValidationErrorBackground, inputValidationErrorBorder, inputValidationErrorForeground, scrollbarSliderActiveBackground, scrollbarSliderBackground, scrollbarSliderHoverBackground, textLinkActiveForeground, textLinkForeground } from 'vs/platform/theme/common/colorRegistry';
+import { IColorTheme, IThemeService } from 'vs/platform/theme/common/themeService';
+import { IWorkspaceTrustManagementService } from 'vs/platform/workspace/common/workspaceTrust';
+import { SIDE_BAR_BACKGROUND } from 'vs/workbench/common/theme';
+import { IWorkbenchAssignmentService } from 'vs/workbench/services/assignment/common/assignmentService';
+import { IAuthenticationService } from 'vs/workbench/services/authentication/common/authentication';
+import { IWorkbenchExtensionEnablementService } from 'vs/workbench/services/extensionManagement/common/extensionManagement';
+import { IIntegrityService } from 'vs/workbench/services/integrity/common/integrity';
 import { IWorkbenchIssueService } from 'vs/workbench/services/issue/common/issue';
+import { IExtensionService } from 'vs/workbench/services/extensions/common/extensions';
+
 
 export class WebIssueService implements IWorkbenchIssueService {
```

Comment:
Hm, it feels strange now that the issue service lives outside the `contrib` layer but depends on the fact that `contrib/issue` exists. I think we should move this service into `contrib` as well, but we would have to see if anyone depends on this service that is not in `contrib` and then potentially move that as well.

Reply:
there is this one that depends on `IWorkebenchIssueService` as well that breaks layering issues. moving it leads to some issues too (not sure if this means we have to move the entire extension management service or just leave as is...)

https://github.com/justschen/vscode/blob/232a7789cf17b677510d7fbc5c9c43984816f583/src/vs/workbench/services/extensionManagement/browser/extensionBisect.ts#L23-L24